dvc/objects/db/base.py

```diff
@@ -29,6 +29,23 @@ def __init__(self, fs, path_info, **config):
         self.cache_types = config.get("type") or copy(self.DEFAULT_CACHE_TYPES)
         self.cache_type_confirmed = False
         self.slow_link_warning = config.get("slow_link_warning", True)
+        self.tmp_dir = config.get("tmp_dir")
```

Comment:
tmp_dir isn't actually used anywhere in ODB yet, but it will be once the current remote index is moved out of Remote into ODB.

In the meantime, this is needed to be able to construct a Remote object from an ODB instance.